articles/quickstart/webapp/nextjs/01-login.md

```diff
@@ -23,98 +23,138 @@ useCase: quickstart
 
-Create the file `app/layout.jsx` as follows:
+### Add the dynamic API route handler
 
-```jsx
-// app/layout.jsx
-import { UserProvider } from '@auth0/nextjs-auth0/client';
+Create a file at `app/api/shows/route.js`. This is your route Handler file using <a href="https://nextjs.org/docs/app/building-your-application/routing/route-handlers#dynamic-route-segments" target="_blank" rel="noreferrer">Dynamic Route Segment</a>. The file declares a `GET` export to call the `shows()` method from the SDK to create the API routes. import the `handleAuth` method from the SDK and call it from the `GET` export.
 
-export default function RootLayout({ children }) {
-  return (
-    <html lang="en">
-    <UserProvider>
-      <body>{children}</body>
-    </UserProvider>
-    </html>
-  );
-}
+```javascript
+// app/api/shows/route.js
+import { NextResponse } from 'next/server';
+import { auth0 } from '../../../lib/auth0';
+
+export const GET = async function shows() {
+  try {
+    const session = await auth0.getSession();
+
+    if (!session) {
+      return NextResponse.json(
+        { error: 'Not authenticated' },
+        { status: 401 }
+      );
+    }
+
+    const res = new NextResponse();
+    const { token: accessToken } = await auth0.getAccessToken();
+    const apiPort = process.env.API_PORT || 3001;
+    const response = await fetch(`http://localhost:${apiPort}/api/shows`, {
+      headers: {
+        Authorization: `Bearer ${accessToken}`
+      }
+    });
+    const shows = await response.json();
+
+    return NextResponse.json(shows, res);
+  } catch (error) {
+    return NextResponse.json({ error: error.message }, { status: error.status || 500 });
+  }
+};
 ```
 
-The authentication state exposed by `UserProvider` can be accessed in any Client Component using the `useUser()` hook.
+Upon execution the following routes for your customers are available:
 
-:::panel Checkpoint
-Now that you have added the dynamic route and `UserProvider`, run your application to verify that your application is not throwing any errors related to Auth0.
-:::
+- `/auth/login`: The route to perform login with Auth0.
+- `/auth/logout`: The route to log the user out.
+- `/auth/callback`: The route Auth0 will redirect the user to after a successful login.
+- `/auth/profile`: The route to fetch the user profile.
+- `/auth/access-token`: The route to verify the user's session and return an access token (which automatically refreshes if a refresh token is available).
+- `/auth/backchannell-logout`: The route to receive a logout_token when a configured Back-Channel Logout initiator occurs.
+
```

Comment:
Most of these are sentence fragments and don't need punctuation. Under /backchannell - make sure logout_token is in code font: `logout_token`